packages/ontario-design-system-component-library/src/components/ontario-header/ontario-header.scss

```diff
@@ -14,8 +14,8 @@
 @use '@ongov/ontario-design-system-global-styles/dist/styles/scss/6-components/_text-inputs.component.scss';
 @use '@ongov/ontario-design-system-global-styles/dist/styles/scss/7-overrides/_visibility.overrides.scss';
 
-$ontario-search-input-padding: 7.2rem;
-$ontario-search-input-padding--mobile: 6.4rem;
+$ontario-search-padding: 7.2rem;
+$ontario-search-padding--mobile: 6.4rem;
 
 $ontario-header-logo-width: 180px;
 $ontario-header-logo-width--mobile: 45px;
```

Comment:
It looks like these escaped being converted to `rem`.

Reply:
There are several spots that still use px. I would like to avoid making those changes within this PR though.

Reply:
Agreed, these should be collected all at once.